samza-core/src/main/java/org/apache/samza/storage/StorageManagerUtil.java

```diff
@@ -155,7 +190,15 @@ public static void writeOffsetFile(File storeBaseDir, String storeName, TaskName
    * @param taskName the task name which is referencing the store
    */
   public static void deleteOffsetFile(File storeBaseDir, String storeName, TaskName taskName) {
-    File offsetFile = new File(getStorePartitionDir(storeBaseDir, storeName, taskName, TaskMode.Active), OFFSET_FILE_NAME);
+    deleteOffsetFile(storeBaseDir, storeName, taskName, OFFSET_FILE_NAME_NEW);
```

Comment:
Do you need to delete the side-inputs offset file too?

Reply:
no, this is only valid for task-stores